core/src/test/java/org/elasticsearch/index/mapper/RangeFieldMapperTests.java

```diff
@@ -189,7 +198,13 @@ protected void doTestStore(String type) throws Exception {
         assertEquals(2, pointField.fieldType().pointDimensionCount());
         IndexableField storedField = fields[1];
         assertTrue(storedField.fieldType().stored());
-        assertThat(storedField.stringValue(), containsString(type.equals("date_range") ? "1477872000000" : "5"));
+        String strVal = "5";
+        if (type.equals("date_range")) {
+            strVal = "1477872000000";
+        } else if (type.equals("ip_range")) {
+            strVal = "/192.168.1.7:/2001:db8:0:0:0:0:0:0";
```

Comment:
I guess we get this by doing a toString on an InetAddress. Could you make sure we use `InetAddresses.toAddrString` instead?